web/e2e/a11y-screenshots/results.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Please, can we remove this file?

Reply:
Yep will do, good spot - this shouldn't be here